### PR TITLE
#140 band website

### PR DESCRIPTION
**Pull request by jugeshraghav**
### Description

It is a multi pager Band Website with simple but user-friendly design.

### Checklist

- [x] I've been assigned an issue related to this PR.
- [ ] I've used beautifiers.
- [x] I've added my Project's name and description to `Index.md`
- [x] I've made a README.md file for my Project.
- [x] The README.md file of my project contains Project title, Description, Use of project, Set up, Stack used and Output (Screenshots). 

## Related Issues or Pull Requests number

#140

### CODE REVIEW

**Comment by jugeshraghav** [comment]
hey @chicken-biryani  @harshita2216 i have added the website please have a look

**Comment by jugeshraghav** [comment]
everything up to date.

File: Index.md [hunk 1]
```diff
@@ -83,4 +83,6 @@
 | [Parallex Website](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/Chess-Game)            | A basic website using HTML, CSS, JAVASCRIPT with modern look. |
 | [Weight Converter](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/Weight%20Converter)              |A web page where used can convert weight from kilograms to grams, ounces and pounds.
 
-| [Restaurant website](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/Restaurant-website)              |A Restuarant website with a simple and user friendly design ad a database linked to it.
\ No newline at end of file
+| [Restaurant website](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/Restaurant-website)              |A Restuarant website with a simple and user friendly design ad a database linked to it.
+
+| [Band website](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/Band Website)              |A Band website with a simple and user friendly design and multiple pages with some extra features.
```
Comment (Ayushparikh-code) [inline Index.md:88]: correct it and update properly!

**Review by Ayushparikh-code — CHANGES_REQUESTED** [review]

**Review by harshita2216 — CHANGES_REQUESTED** [review]
Hey, the images in the readme file aren't working I think and the link of your project is wrongly written in the index.md file. Instead of space, %20 should be written.
Please make the above changes. Thanks!

**Comment by jugeshraghav** [comment]
i have corrected the name of the website in the index,md and the images are visible in the readme at my end please check again.
Screenshot (132)

**Review by harshita2216 — APPROVED** [review]

**Pull request merged**